views/components/flexible-content/macro.njk

```diff
@@ -14,11 +14,22 @@
             isReversed = cycler.next() if cycler else false
         ) }}
     {% elseif part.type === 'relatedContent' %}
-        {{ relatedContent(
-            articles = part.content,
-            title = part.heading,
-            usePromo = true
-        ) }}
+        <h2 class="t--underline">{{ part.heading }}</h2>
+        <ul class="flex-grid flex-grid--3up">
+            {% for article in part.content %}
+                <li class="flex-grid__item">
+                    {{ promoCard({
+                        "title": article.title + "floop",
```

Comment:
<img width="903" alt="image" src="https://user-images.githubusercontent.com/394376/77406974-5a4d3300-6dad-11ea-8776-d0efaffc90fe.png">

Reply:
birthdayfloop

Reply:
Lotteryfloop

Reply:
Also those bullet points look like a regression, there's some gnarly CSS to fix that (`.related-content .related-content__item`)

Reply:
Hmm which page type was that on?

Reply:
The 25th fund page: `/about/the-national-lotterys-25th-birthday`

Reply:
Worked it out, this page was using the _old_ flexible content template, where the flex-grid was still inside a `s-prose` area. Made a root cause fix by resetting the list-style and margin explicitly, should be pretty safe but worth doing a scan of the site to confirm this doesn't cause any regressions.